jax/_src/numpy/lax_numpy.py

```diff
@@ -467,16 +467,16 @@ def result_type(*args: Any) -> DType:
 
 @jit
 def trunc(x: ArrayLike) -> Array:
-  """Return the element-wise truncated value of the input.
+  """Return truncated value of the input to the nearest integer towards zero.
```

Comment:
I think `Round input to the nearest integer towards zero.` would be far more clear here.

Reply:
Modified as suggested